templates/_helpers.tpl

```diff
@@ -353,6 +353,21 @@ Sets extra injector service annotations
   {{- end }}
 {{- end -}}
 
+{{/*
+Sets extra injector webhook annotations
+*/}}
+{{- define "injector.webhookAnnotations" -}}
+  {{- if .Values.injector.webhookAnnotations }}
+  annotations:
+    {{- $tp := typeOf .Values.injector.webhookAnnotations }}
+    {{- if eq $tp "string" }}
```

Comment:
Is it worth supporting string for new options?

Reply:
Good question, though I feel like even with new options we need to keep supporting both types for consistency with the other annotation options in the chart.